### PR TITLE
Naive maps updates

### PR DESCRIPTION
**Pull request by petiay**
Added several options:
- Can now calculate the median of the BEAST results (instead of by default just the mean)
- Can make maps of three stellar parameters - for now they are log(T), log(age), and initial Mass. Easy to tweak or add more.
- Can input a chi2min threshold to cut BEAST stats before making maps
- Updated map file naming (from experience it's nice to know the map resolution at a glance esp. when there are comparisons with various ISM maps).

### CODE REVIEW

File: megabeast/tools/make_naive_maps.py [hunk 1]
```diff
@@ -79,7 +90,8 @@ def create_naive_maps(stats_filename, pix_size=10.0, verbose=False):
     for i in range(n_x + 1):
         for j in range(n_y + 1):
             (tindxs,) = np.where((x == i) & (y == j))
-            # tindxs, = np.where((x == i) & (y == j) & (cat['chi2min'] < 10.))
+            if chi2mincut:
+                (tindxs,) = np.where((x == i) & (y == j) & (cat['chi2min'] < chi2mincut))
```
Comment (karllark) [inline megabeast/tools/make_naive_maps.py:94]: Would be better if line 92 was in an else statement after line 94.  Then there wouldn't be a double execution.

File: megabeast/tools/make_naive_maps.py [hunk 2]
```diff
@@ -88,14 +100,18 @@ def create_naive_maps(stats_filename, pix_size=10.0, verbose=False):
                     values = cat[cur_stat + "_" + stat_type][tindxs]
                     values_foreach_pixel[cur_stat][i, j] = values
                     summary_stats[j, i, k] = np.average(values)
-                    summary_sigmas[j, i, k] = np.std(values, ddof=1) / math.sqrt(
-                        len(values)
-                    )
+                    summary_sigmas[j, i, k] = np.std(values, ddof=1) / math.sqrt(len(values))
+                    if median:
+                        summary_stats[j, i, k] = np.median(values)
+                        summary_sigmas[j, i, k] = median_abs_deviation(values)
```
Comment (karllark) [inline megabeast/tools/make_naive_maps.py:106]: What the change to "median_abs_deviation"?  This is different than the np.std. 

In addition, line 103 computes the standard deviation of the mean and "median_abs_deviation" computes the equivalent of the standard deviation.  Feels like the "median_abs_deviation" value needs to be divided by the same number as line 103.

**Review by karllark — COMMENTED** [review]
Overall looks good.  Have a couple of comments that should be addressed.

File: megabeast/tools/make_naive_maps.py (hunk 1 above)
Comment (petiay) [reply megabeast/tools/make_naive_maps.py]: Addressed with commit from May 23.

File: megabeast/tools/make_naive_maps.py (hunk 2 above)
Comment (petiay) [reply megabeast/tools/make_naive_maps.py]: Addressed - see updates.

**Review by karllark — APPROVED** [review]
Looks good to me.

**Comment by karllark** [comment]
No tests to speak of for megabeast yet.  So ignoring these for now.

**Pull request merged**